terraform/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+berkeley_subnets      = [ "128.32.0.0/16", "136.152.0.0/16", "169.229.0.0/16"]
+
+berkeley_subnets6     = [ "2607:f140::/32" ]
+
+decal_ssh_fingerprint =  ["e5:95:8d:ac:35:81:f3:ec:9a:57:2a:23:b8:2f:19:7c" ]
```

Comment:
I think these could just be put into the `decal-vms.tf` file with the appropriate `variable` declarations as defaults? Not sure if that's better, but might make this a bit cleaner since then you wouldn't need multiple files, and I don't think it really makes it harder to work with or maintain since it'll be clear where the variables come from then.

Reply:
This is all layout/stuff adapted directly from the Terraform getting started guide, so it shouldn't be a big deal to adapt to fit our workflow.